.buildkite/pipeline.yml

```diff
@@ -151,9 +145,9 @@ steps:
     key: packaging
     if: build.branch != "autodoc/master" && build.source != "schedule"
     commands:
-    - nix-build release.nix -A static -o tzbtc-static
-    - nix-build release.nix -A deb -o tzbtc-client-deb
-    - nix-build release.nix -A rpm -o tzbtc-client-rpm
+    - nix-build -A release.static -o tzbtc-static
+    - nix-build -A releaes.deb -o tzbtc-client-deb
+    - nix-build -A release.rpm -o tzbtc-client-rpm
```

Comment:
Why not make these bona fide flake outputs instead of using non-standard `release` attribute?